composer.json

```diff
@@ -29,7 +29,9 @@
     },
     "require-dev": {
         "sensio/generator-bundle": "~2.3",
-        "phpunit/phpunit": "~4.3"
+        "phpunit/phpunit": "~4.3",
+        "ongr/ongr-strict-standard": "~1.0-beta",
+        "squizlabs/php_codesniffer": "~1"
```

Comment:
I'm not sure we are ready for code sniffer 2.0. I would say just add ~1.4 for now. @tautrimas what do you think?